templates/al2023/variables-default.json

```diff
@@ -26,7 +26,7 @@
     "remote_folder": "/tmp",
     "runc_version": "*",
     "security_group_id": "",
-    "source_ami_filter_name": "al2023-ami-minimal-2023.*-kernel-6.12-*",
+    "source_ami_filter_name": "al2023-ami-minimal-2023.*-kernel-6.1-*",
```

Comment:
chatted offline: it's going to take a week or two for support to land in all our other dependencies.